programs-remote-listings/templates/shortcode-programs.php

```diff
@@ -21,11 +21,6 @@
                 <div class="rs-program-thumbnail"><a href="<?php echo $details_url; ?>"><img src="<?php echo $program_image_url; ?>"></a></div>
             <?php endif; ?>
 
-            <?php if ($program->teacher_details->teacher_objects && ! empty($show_first_teacher_photo)) : ?>
```

Comment:
as it turns out, we don't pass teacher_details and price_first in _program_base_ or _program_registration_